pkgs/native_assets_cli/example/use_dart_api/README.md

```diff
@@ -0,0 +1,8 @@
+An example that uses `dart_api_dl.h`.
```

Comment:
Expand this more, so people like me don't have to start looking what `dart_api_dl.h` is in the first place, and why one would want to use it.